src/airpyllution/airpyllution.py

```diff
@@ -146,13 +144,13 @@ def get_air_pollution(lat, lon, api_key, fig_title=""):
     try:
         response = requests.get(url=url, params=params)
         response_obj = response.json()
-        data = convert_data_to_pandas(response_obj)
+        try:
+            data = convert_data_to_pandas(response_obj)
+        except:
+            if "cod" in response_obj:
+                return response_obj["message"]
     except:
-        response = requests.get(url=url, params=params)
```

Comment:
@danfke i've just refactored this so that we're not repeating the requests api call again in the `except` case. Instead we have a nested try-except. The outer try-except deals with failing API call, the inner ones handles bad responses from the API - i.e if the `conver_data_to_pandas fails`